docs/stepbystep.md

```diff
@@ -397,15 +418,15 @@ We will get something like this:
 
 That shows the information we sent with the measures has been written to the Context Broker properly.
 
-## <a name="acls">Using ACLs to secure provisioning access</a>
+## Using ACLs to secure provisioning access
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated